### DISCUSSION TITLE
Posthtml plugin works only when using .posthtmlrc.config.js.

### DISCUSSION THREAD

**Discussion by Venikeee1** [parcel-bundler/parcel#8364] [Q&A] [answered]
I am using `posthtml-include` to include html partials. According to doc it is better to use  `.posthtmlrc` in json format but in this way it doesn't work. Problem was solved by using `.posthtmlrc.config.js`. Question is: is it possible to use `.posthtmlrc` insteed of `.posthtmlrc.config.js`? Here is the example https://codesandbox.io/s/parcel-issue-y68xwd?file=/index.html

**Comment by mischnic** [comment, marked as answer]
These are the config file name that should be supported by Parcel:
https://github.com/parcel-bundler/parcel/blob/9e5d05586577e89991ccf90400f2c741dca11aa3/packages/transformers/posthtml/src/PostHTMLTransformer.js#L20-L31

(And you were running Parcel locally? Codesandbox doesn't actually use Parcel)

**Reply by Venikeee1** [reply]
I didn't know that codesandbox doesn't work properly with parcel. 
 Yes I've done it locally and it doesn't work with `.posthtmlrc` but everything fine with `posthtml.config.js`. Also one of my friend face the same issue

**Reply by mischnic** [reply]
Both appear to be working for me locally with this:
[parcel-issue 2.zip](https://github.com/parcel-bundler/parcel/files/9261894/parcel-issue.2.zip)

**Reply by Venikeee1** [reply]
First of all thank you for fast response. That was my bad I didn't check parcels version. After upgrading it to the latest everything works fine.
